src/libical-glib/i-cal-object.c.in

```diff
@@ -341,8 +341,9 @@ i_cal_object_construct(GType object_type,
 
     g_return_val_if_fail(object_type != G_TYPE_INVALID, NULL);
     g_return_val_if_fail(native != NULL, NULL);
-    if (owner)
+    if (owner) {
         g_return_val_if_fail(G_IS_OBJECT(owner), NULL);
+    }
```

Comment:
I've seen lot of occurrences of this in GLib projects:
```suggestion
    g_return_val_if_fail(!owner || G_IS_OBJECT(owner), NULL);
```

Reply:
It's just a different way to check the same, with less output on terminal when not using the suggestion :)